index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
Pls make sure to add the `<title>` tag inside the `<head>` tag, as it follows best practices, semantic rules and also if you have noticed this gives a linter error too. Kindly fix this.